qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/AdministrationPage.java

```diff
@@ -34,32 +35,37 @@ public String getPageUrl() {
 	}
 
 	public ManageUserPage clickOnManageUsers() {
-		findElement(MANAGE_USERS).click();
+		findElement(MANAGE_USERS_LINK).click();
```

Comment:
i think MANAGE_USERS  is the best  to use. cc @kdaud what do you think

Reply:
> @kdaud what do you think

It would be a good ideal to have the variable names for the page elements to identify whether the element is a button, link, icon or field.